Makefile

```diff
@@ -0,0 +1,15 @@
+clean: test server
+	@echo "\n\n\nShutting down the server\n"
```

Comment:
Here is a good example:
https://github.com/Stratoscale/upseto/blob/master/Makefile